### ISSUE TITLE
[FEAT] Version in executable filename

### ISSUE THREAD

**Issue by Kochise** [spyder-ide/spyder#15023]
Put the version in the released executable filename, for 4.2.5 :

Spyder_4.2.5-Lite.dmg
Spyder_4.2.5.dmg
Spyder_4.2.5_64bit_full.exe
Spyder_4.2.5_64bit_lite.exe

That would help since the version number is not even present in the PE details.

Regards.

**Comment by juanis2112** [comment]
Hi @Kochise thanks for opening this. This sounds like a good idea, what do you think @ccordoba12, @dalthviz?

**Comment by dalthviz** [comment]
Hi @Kochise I think that could be nice +1 from my side :+1:

**Comment by Kochise** [comment]
Any news about this request ?

**Comment by ccordoba12** [comment]
I disagree @dalthviz. I think it should naming should be `Spyder-x.x.x_64bit_lite.exe`

**Comment by ccordoba12** [comment]
@Kochise, just to clarify, this is very very low priority for us, so it could take us a several months or even year to implement (unless you can give us a hand). Sorry but there are many more pressing issues to address for now.

**Comment by Kochise** [comment]
Ok, I haven't understood it would require such a massive change in a CI/CD to insert the release version into the artifact names.

**Comment by ccordoba12** [comment]
It's not that the change is massive (I don't know if you're being sarcastic here, but if you do, please refrain comments like that in the future). It's simply that we have a lot of other things to do and this is something that it's not breaking functionality or something like that.
